docs/api-reference/widgets/fullscreen-widget.md

```diff
@@ -0,0 +1,68 @@
+# FullscreenWidget
+
+## Variables
```

Comment:
These variables are intended for all core widgets to use. For now, documenting in FullscreenWidget since it's the only usage.